definitions/contested/json/ComplexTypes/DFR-3332-nonprod.json

```diff
@@ -82,6 +92,15 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "29/10/2024",
+    "ID": "FR_judgeApproval",
+    "ListElementCode": "amendedDocument",
+    "FieldType": "Document",
+    "ElementLabel": "You must upload Microsoft Word documents. Document names should clearly reflect the party name, the type of hearing and the date of the hearing. For example “JonesFDA11Jul24”",
```

Comment:
Are we enforcing this file type restriction in the CCD config?

Reply:
[502b231](https://github.com/hmcts/finrem-ccd-definitions/pull/2138/commits/502b23132c1ca73e8db607f87988d4edda7d40df)